### ISSUE TITLE
.print_deck_info() doesn't populate Serial or Firmware Version on Stream Deck XL

### ISSUE THREAD

**Issue by PFython** [abcminiuser/python-elgato-streamdeck#21]
Running `example_deckinfo.py ` on a new Stream Deck XL (just arrived this week), I get:

Deck 0 - Stream Deck XL.
         - ID: b'\\\\?\\hid#vid_0fd9&pid_006c#7&a8de7c&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}'        
         - Serial: ''
         - Firmware Version: ''
         - Key Count: 32 (8x4 grid)
         - Key Images: 96x96 pixels, JPEG format, rotated 0 degrees, mirrored horizontally/vertically

**Comment by abcminiuser** [comment]
Thanks for the report. Not owning the hardware is a bit of a pain point for compatibility - so I hope you're willing to test out a fix!

I've pushed a potential fix in 059c5c7f7b03da969dc18b6075e95fd872c69327. Would you mind trying that on your device please?

**Comment by PFython** [comment]
Looks good!  Maybe just tweak the formatting to remove trailing spaces and` (' `or `W'` ?

`.split(" ")[0]` should do the trick I think.

```
Found 1 Stream Deck(s).

Deck 0 - Stream Deck XL.
         - ID: b'\\\\?\\hid#vid_0fd9&pid_006c#7&a8de7c&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}'
         - Serial: 'CL30I1A03586                  ('   
         - Firmware Version: '1.00.006
 W'
         - Key Count: 32 (4x8 grid)
         - Key Images: 96x96 pixels, JPEG format, rotated 0 degrees, mirrored horizontally/vertically
```

**Comment by abcminiuser** [comment]
I suspect the extra padding bytes will be zeros (NUL) characters rather than spaces, as it would make sense on the embedded side.

I've patched this in 324d61e73f6a50b5304953e530b5c30d4faa6150 to trim both zeros and actual spaces from the end of the string just in case. Please test when you get the chance and if it works I'll tag off for 0.5.1.

**Comment by PFython** [comment]
Yep, serial and firmware display beautifully on StreamDeck XL now.  Thanks!

**Comment by abcminiuser** [comment]
Released as 0.5.1. Thanks for the assistance!
